content/posts/matplotlib/emoji-mosaic-art/index.md

```diff
@@ -22,7 +22,7 @@ So instead of going through the process of installing Rust, I decided to take th
 
 In this post, I'll try to explain my process as we attempt to recreate similar mosaics as this one below. I've aimed this post at people who've worked with _some_ sort of image data before; but really, anyone can follow along.
 
-![alt text](./warhol.png "Emosaic by Will Dady")
+![alt text ](./warhol.png "Emosaic by Will Dady")
```

Comment:
```suggestion
![A 3 X 3 mosaic that has images of people created using emojis.](./warhol.png "Emosaic by Will Dady")
```

Reply:
Would be nice to have a review on this one too @isabela-pf ! I'm not sure if I was able to describe this image appropriately - it feels tricky!

Reply:
I agree it's tricky because it's referencing another piece of art! The cool thing is that the author left us what I'm guessing is supposed to be alt text in the quotes. `"Emosaic by Will Dady"` describes the image, but markdown doesn't render alt text from quotes in the link. 

When I find things like this, I like to integrate the original attempted alt text since the author thought it was important. On top of that, I add the art work's name, artist, date (if it hasn't been added elsewhere yet) so that people can look it up if they want. Whether or not I describe the art usually depends on the context of post, but in this case the author doesn't reference it beyond it being an example of emosiac art.